src/utils/markdown.rs

```diff
@@ -124,7 +124,7 @@ pub fn escape_code(s: &str) -> String {
 pub fn user_mention_or_link(user: &User) -> String {
     match user.mention() {
         Some(mention) => mention,
-        None => link(user.url().as_str(), &user.full_name()),
+        None => link(user.url().as_str(), &escape(&user.full_name())),
```

Comment:
This escape should be inside `link`:
```diff
-    format!("[{}]({})", text, escape_link_url(url))
+    format!("[{}]({})", escape(text), escape_link_url(url))
```

Reply:
Actually, no, I take that back. `link("*a*", "https://example.com")` is a valid thing, so we should not autoescape text in `link`...